app/views/granules/prepare_script.html.erb

```diff
@@ -43,7 +43,13 @@
 <body>
 <div id="errors"></div>
 <div id="loading">Preparing download script. Fetching downloadable granule links...</div>
-<div id="download-description" style="display: none;">Script preparation completed. Please click the button to download the script. <a href="#" id="download-button">Download Script File</a></div>
+<div id="download-description" style="display: none;">
+  <h2>How to use this script</h2>
+  <p>Linux: You must first make the script an executable by running the line 'chmod 777 download.sh' from the command line.  After that is complete, the file can be executed by typing './download.sh'.</p>
+  <p>Windows: The file can be executed within Windows by first installing a Unix-like command line utility such as <a href='https://www.cygwin.com/'>Cygwin</a>. After installing Cygwin (or a similar utility), run the line 'chmod 777 download.sh' from the utility's command line, and then execute by typing './download.sh'.</p>
+  <p>For a detailed walk through of this process, please reference this <a target="_blank" href='https://wiki.earthdata.nasa.gov/display/EDSC/How+To%3A+Use+the+Download+Access+Script'>How To guide</a>.</p>
```

Comment:
Sorry to nitpick... but I'd prefer this paragraph to be right after the Linux instructions. As it is, it could be read that the detailed instructions are for how to accomplish this using Windows... which it's not.

Reply:
@mreese84 done - happy to keep fiddling with it if it needs more tweaking!  Just let me know.